### PR TITLE
model zoo

### PR DESCRIPTION
**Pull request by ofrimasad**

### CODE REVIEW

File: documentation/source/model_zoo.md [hunk 1]
```diff
@@ -77,3 +78,46 @@ All the available models are listed in the column `Model name`.
...
+- [VGG](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/classification_models/vgg.py) - Very Deep Convolutional Networks for Large-scale Image Recognition [https://arxiv.org/pdf/1409.1556](https://arxiv.org/pdf/1409.1556)
+  
+  
+### Object Detection
+  
+- [CSP DarkNet](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/csp_darknet53.py)
+- [DarkNet-53](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/darknet53.py)
+- [SSD (Single Shot Detector)](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/ssd.py) - [https://arxiv.org/pdf/1512.02325](https://arxiv.org/pdf/1512.02325)
+- [YOLOX](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/yolox.py) - [https://arxiv.org/abs/2107.08430](https://arxiv.org/abs/2107.08430)
+
```
Comment (BloodAxe) [inline documentation/source/model_zoo.md:113]: PPYoloE missing?

File: documentation/source/model_zoo.md [hunk 2]
```diff
@@ -77,3 +78,46 @@ All the available models are listed in the column `Model name`.
...
+- [LadderNet](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/segmentation_models/laddernet.py) - Multi-path networks based on U-Net for medical image segmentation [https://arxiv.org/pdf/1810.07810](https://arxiv.org/pdf/1810.07810)
+- [RegSeg](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/segmentation_models/regseg.py) - Rethink Dilated Convolution for Real-time Semantic Segmentation [https://arxiv.org/pdf/2111.09957](https://arxiv.org/pdf/2111.09957)
+- [ShelfNet](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/segmentation_models/shelfnet.py) - [https://arxiv.org/pdf/1811.11254](https://arxiv.org/pdf/1811.11254)
+- [STDC](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/segmentation_models/stdc.py) - Rethinking BiSeNet For Real-time Semantic Segmentation [https://arxiv.org/pdf/2104.13188](https://arxiv.org/pdf/2104.13188)
+  
+</details>
```
Comment (shaydeci) [inline documentation/source/model_zoo.md:123]: Also pose estimation

**Review by shaydeci — CHANGES_REQUESTED** [review]

File: documentation/source/model_zoo.md (hunk 2 above)
Comment (ofrimasad) [reply documentation/source/model_zoo.md:143]: added

File: documentation/source/model_zoo.md (hunk 1 above)
Comment (ofrimasad) [reply documentation/source/model_zoo.md:129]: added

**Review by BloodAxe — APPROVED** [review]

**Review by Louis-Dupont — APPROVED** [review]
LGTM

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**